src/Microsoft.Data/BaseColumn.cs

```diff
@@ -93,8 +93,20 @@ public void SetName(string newName, DataFrame dataFrame = null)
 
         public virtual Dictionary<TKey, ICollection<long>> GroupColumnValues<TKey>() => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns a DataFrame containing counts of unique values
+        /// </summary>
+        public virtual DataFrame ValueCounts() => throw new NotImplementedException();
+
         public virtual GroupBy GroupBy(int columnIndex, DataFrame parent) => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns a new column with nulls replaced by value
+        /// </summary>
+        /// <remarks>Tries to convert value to the column's DataType</remarks>
+        /// <param name="value"></param>
+        public virtual BaseColumn FillNulls(object value, bool inPlace = false) => throw new NotImplementedException();
```

Comment:
I don't see this method implemented for `StringColumn`.  Also, I would think that `ArrowStringColumn` should override this and throw a `NotSupportedException` instead.